RELEASE_NOTES.md

```diff
@@ -1,3 +1,12 @@
+VERSION 0.2.0 (Released TDB)
+----------------------------
+
+BACKWARD INCOMPATIBILIES:
+
+- Changes to make the client fully compatible with the Blobstore
+  - getFormat() has been moved from ShockFileInformation to ShockNode.
```

Comment:
Sooo... it turns out that the blobstore has had the `format` key in the wrong place for 5+ years and nobody noticed. Oops.